scheduler_plugins/slurm_plugin/artifacts/handlers/head_finalize.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+#
+# Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+# with the License. A copy of the License is located at http://aws.amazon.com/apache2.0/
+# or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+# OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+set -e
+
+sudo -E cinc-client -z -o slurm_plugin_cookbook::head_node_finalize --log-level info --force-formatter
```

Comment:
for recipe names, what about using the convention <phase>-<node-type or name>, e.g.
finalize_head_node
init_compute
install_slurm
etc...